app/register/page.test.tsx

```diff
@@ -102,4 +107,40 @@ describe('Register', () => {
 
     mockUseAuthContext.isSignedIn = false;
   });
+
+  test('successfully registers and shows success notification', async () => {
```

Comment:
Same with these test. This is incorrectly tested. OnSubmit, it's either going to pass or fail. You need to mock the response of a success or failure. Then, the alert will show up within the component. Check that the alert is in the component and we're expecting the correct variant and message.